client/src/components/App.js

```diff
@@ -22,6 +22,7 @@ const App = (props) => {
         setCurrentUser(null)
       })
   }, [])
+  
   return (
     <Router>
       <TopBar user={currentUser} />
```

Comment:
refer to my auth clinic on passing currentUser down to your TripShow, and then to your comment form. Below should work though
```
<Route exact path="/:id">
   <TripShow user={currentUser}
<Route>
```

```